### PR TITLE
added some hints if you are using google colab

### PR DESCRIPTION
**Pull request by GriffGreen**

### CODE REVIEW

File: tutorials/robot-marbles-part-1/robot-marbles-part-1.ipynb [hunk 1]
```diff
@@ -227,7 +229,7 @@
    "metadata": {},
    "source": [
     "# Running the engine\n",
-    "We are now ready to run the engine with the configuration defined above. Instantiate an ExecutionMode, an ExecutionContext and an Executor objects, passing the Configuration object to the latter. Then run the `execute()` method of the Executor object, which returns the results of the experiment in the first element of a tuple."
+    "We are now ready to run the engine with the configuration defined above. Instantiate an ExecutionMode, an ExecutionContext and an Executor objects, passing the Configuration object to the latter. Then run the `execute()` method of the Executor object, which returns the results of the experiment in the first element of a tuple. In Google Colab, you will want to start a new execution by hitting the `+Code` button in the upper left."
```
[the remark this answers is not in the document]
Comment (GriffGreen) [reply tutorials/robot-marbles-part-1/robot-marbles-part-1.ipynb]: I ran into having trouble figuring out how to add a new cell in google colab... I probably have the language wrong... Let me try to fix it.

**Comment by GriffGreen** [comment]
I think i changed the wording @markusbkoch This should probably be closed tho since there was a new rewrite?

**Comment by JEJodesty** [comment]
@GriffGreen
Are you sure?

**Comment by markusbkoch** [comment]
> @GriffGreen
> Are you sure?

Yes, I think Griff is right - we should close this PR and deal with all things related to Google Colab in the demos repo (eg https://github.com/cadCAD-org/demos/issues/3). 

Once we merge https://github.com/cadCAD-org/demos/pull/2 (I'm working on it) we should be able to reopen this same PR there.

**Comment by JEJodesty** [comment]
@GriffGreen 
Thanks for your contribution!
tutorials are now in another repo.
please re-submit your PR here: https://github.com/cadCAD-org/demos

**Pull request closed without merging**